src/pages/stdlib/{StdlibSection.slug}.js

```diff
@@ -0,0 +1,119 @@
+import * as React from "react"
+import Layout from "../../components/layout"
+import { graphql } from "gatsby"
+import SidebarToc from "../../components/stdlib-toc";
+import {useEffect} from "react";
+
+import Prism from "prismjs";
+import "prismjs/themes/prism-tomorrow.css";
+import nickelLanguageDefinition from "../../prism/nickel";
+
+import ReactMarkdown from 'react-markdown';
+
+const Stdlib = ({data}) => {
+    const object = JSON.parse(data.stdlibSection.internal.content);
+    const slug = data.stdlibSection.slug; 
+    const name = data.stdlibSection.name; 
```

Comment:
Nitpicky nitpick: I think we can write this in one line with `const {slug, name} = data.stdlibSection;`

Reply:
Nice 👍